ui/apps/platform/src/Components/CompoundSearchFilter/utils/imageFilterConfig.ts

```diff
@@ -0,0 +1,91 @@
+import { CompoundSearchFilterConfig, DeepRequired } from '../types';
+
+// We want to only retrieve the "Image" config and make sure all the "attributes" are required (not optional)
+export type ImageCompoundSearchFilterConfig = DeepRequired<
+    Required<Pick<CompoundSearchFilterConfig, 'Image'>>
+>;
+
+export const imageFilterConfig: ImageCompoundSearchFilterConfig = {
```

Comment:
This could be removed if you create the value first, infer the type from that, and export both as mentioned in a comment above.